tasks/new_e2e_tests.py

```diff
@@ -164,6 +167,23 @@ def run(
             'You can also add `E2E_DEV_MODE="true"` to run in dev mode which will leave the environment up after the tests.'
         )
 
+    if logs_post_processing:
+        post_processed_output = post_process_output(
+            test_res[0].result_json_path, test_depth=logs_post_processing_test_depth
+        )
+
+        os.makedirs(logs_folder, exist_ok=True)
+        write_result_to_log_files(post_processed_output, logs_folder)
+        try:
+            pretty_print_logs(post_processed_output)
+        except TooManyLogsError:
+            print(
+                color_message(
+                    "Too many logs to print, skipping pretty print. You can still find them properly organized in the job artifacts",
+                    "yellow",
+                )
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏I prefer to print `WARNING: <message>` with `WARNING` in yellow and the message as usual

Reply:
Or maybe we can color all the message in bold?